### PR TITLE
examples: Rename the viscoacoustic equations 

### PR DESCRIPTION
**Pull request by nogueirapeterson**
Hi Guys,
This PR only renames the viscoacoustic equations implemented considering their origin (rheological models), rather than the name of authors who used these equations.

### CODE REVIEW

File: examples/seismic/viscoacoustic/operators.py [hunk 1]
```diff
@@ -38,11 +38,15 @@ def src_rec(p, model, geometry, **kwargs):
     return src_term + rec_term
 
 
-def sls_1st_order(model, geometry, p, r=None, **kwargs):
+def SLS_1st_order(model, geometry, p, r=None, **kwargs):
```
Comment (georgebisbas) [inline examples/seismic/viscoacoustic/operators.py:41]: I think it should stay lowercase? @FabioLuporini @mloubout

**Review by georgebisbas — COMMENTED** [review]
Not sure I can have some input on this PR.

File: examples/seismic/viscoacoustic/operators.py (hunk 1 above)
Comment (mloubout) [reply examples/seismic/viscoacoustic/operators.py:41]: yeah should stay lower to be consistent

Comment (nogueirapeterson) [reply examples/seismic/viscoacoustic/operators.py]: okay. I already fixed it.

**Review by mloubout — APPROVED** [review]

**Comment by FabioLuporini** [comment]
Merged, thanks a lot!

**Pull request merged**